src/sql/workbench/contrib/queryplan2/browser/queryPlan.ts

```diff
@@ -0,0 +1,738 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import 'vs/css!./media/queryPlan2';
+import * as azdata from 'azdata';
+import { IPanelView, IPanelTab } from 'sql/base/browser/ui/panel/panel';
+
+import { localize } from 'vs/nls';
+import { dispose } from 'vs/base/common/lifecycle';
+import { IConfigurationRegistry, Extensions as ConfigExtensions } from 'vs/platform/configuration/common/configurationRegistry';
+import { Registry } from 'vs/platform/registry/common/platform';
+import { ActionBar, ActionsOrientation } from 'vs/base/browser/ui/actionbar/actionbar';
+import * as DOM from 'vs/base/browser/dom';
+import { PropertiesAction } from 'sql/workbench/contrib/queryplan2/browser/actions/propertiesAction';
+
+import * as azdataGraphModule from 'azdataGraph';
+let azdataGraph = azdataGraphModule();
```

Comment:
this call is needed since the azdataGraph npm module isn't being packaged following CommonJS format.  This will be fixed in a later PR.